src/modules/ChannelList/components/ChannelPreview/index.tsx

```diff
@@ -227,7 +228,12 @@ const ChannelPreview: React.FC<ChannelPreviewInterface> = ({
         https://github.com/facebook/react/issues/11387#issuecomment-340019419
       */}
       {
-        showMobileLeave && isMobile && (
+        /**
+         * Do not show unread count for focused channel. This is because of the limitation where
+         * isScrollBottom and hasNext states needs to be added globally but they are from channel context
+         * so channel list cannot see them with the current architecture.
+         */
+        (channel.url !== currentChannel?.url) && showMobileLeave && isMobile && (
```

Comment:
In my memory,
we call `markAsRead` when the message list scroll reaches the bottom,
and then the ChannelList receives the `onUnreadMemberCountUpdated` event,
so the unread count will be removed

**BTW, this is a channel leave button, not the UnreadCount badge**

Reply:
The issue with that behaviour is blink like effect. When scroll is at the bottom,  unread count updates twice: 0 -> 1 -> 0. This is blink like effect and looks unnatural and customer complained about it. This is why we removed it.
It is UnreadCount badge. I tested? @HoonBaek